apps/admin-portal/src/components/identity-providers/wizard/steps/wizard-summary.tsx

```diff
@@ -82,26 +82,28 @@ export const WizardSummary: FunctionComponent<WizardSummaryProps> = (
         });
     };
 
-    return (
-        <Grid className="wizard-summary">
-            <Grid.Row>
-                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 } textAlign="center">
-                    <div className="general-details">
-                        <AppAvatar
-                            name={ identityProvider?.name }
-                            image={ identityProvider?.image }
-                            size="tiny"
-                        />
-                        {identityProvider?.name && (
-                            <Heading size="small" className="name">{identityProvider.name}</Heading>
-                        )}
-                        {identityProvider?.description && (
-                            <div className="description">{identityProvider.description}</div>
-                        )}
-                    </div>
-                </Grid.Column>
-            </Grid.Row>
+    const getGeneralDetailsComponent = () => {
+        return <Grid.Row>
```

Comment:
Shall we get the <Grid.Row> to the next line?

Reply:
Fixed in 6311ed7b87d8256e38389b5145f845d46e4c8564.